data/20_elementary.pantheon.wm.gschema.override

```diff
@@ -1,6 +1,7 @@
 [org.gnome.mutter:Pantheon]
 dynamic-workspaces = true
 attach-modal-dialogs = false
-button-layout = 'close:maximize'
 edge-tiling = true
-enable-animations = true
+
+[org.gnome.desktop.wm.preferences]
```

Comment:
This should still be 
```suggestion
[org.gnome.desktop.wm.preferences:Pantheon]
```
right?

Reply:
Yep, you only want that preference on Pantheon (it's probably the most pantheon specific thing). That way it can co-exist with other session with different button-layout